### DISCUSSION TITLE
Can’t login on grafana using sudo salt-call pillar.get secrets password

### DISCUSSION THREAD

**Discussion by yamyamyamyam** [Security-Onion-Solutions/securityonion#7436] [Unsupported Versions]
Hi, can I ask what will be the procedure to log in on grafana using so v. 2.3.100?

username: admin and password from sudo salt-call pillar.get secrets

Thank you very much

**Comment by Acewiza** [comment]
I didn't know there was any "default credentials."  Every SO install I ever did established user-supplied accounts and passwords.

**Reply by yamyamyamyam** [reply]
I used admin and password generated from sudo salt-call pillar.get secrets but didn't work

**Reply by xxvs** [reply]
Secrets aren't the same as passwords. What credentials are you using to log in to SOC? It should be the same for Grafana.

Or check [this](https://docs.securityonion.net/en/2.3/adding-accounts.html?highlight=add%20user#soc) out for adding new users.

**Reply by yamyamyamyam** [reply]
Hi sir, 

I did use the credentials that I make to enter the SOC but it did not work when logging into grafana. 

Ill check the link you gave. Thank you very much

**Comment by dougburks** [comment]
Please change the title of this discussion so that it is not ALL CAPS as this comes across as YELLING!

**Reply by yamyamyamyam** [reply]
Im very sorry for that. I’ll never do it again. 

Thank you
